app/src/androidTest/java/co/nimblehq/compose/crypto/test/MockUtil.kt

```diff
@@ -97,4 +96,9 @@ object MockUtil {
             lastUpdated = "lastUpdated"
         )
     )
+
+    val coinPrices = listOf(
+        CoinPrice(1000,BigDecimal.ZERO),
+        CoinPrice(2000, BigDecimal.ONE)
+    )
```

Comment:
Regarding this https://github.com/nimblehq/jetpack-compose-crypto/pull/95/files#r1111649598 I think we can get rid of this because we don't use it for any assertion, Let's add it later when we need it 🙏 
```suggestion
```

Reply:
@Wadeewee We need it for mocking `mockGetCoinPricesUseCase` or we will get `no answer found for getCoinPrices()`

Reply:
@kaungkhantsoe You can provides like this 🙏 

```kotlin
every { mockGetCoinPricesUseCase.execute(any()) } returns mockk()
```

I have applied on the UnitTest as well see more this [commit](https://github.com/nimblehq/jetpack-compose-crypto/pull/96/commits/70f28eab01dd44ce45eab9c5db8b991935ec9c0c)

Reply:
@Wadeewee I got this error 👇 
<img width="1440" alt="Screenshot 2023-02-21 at 1 48 38 PM" src="https://user-images.githubusercontent.com/32578035/220268592-cbefc416-571a-46ad-a4bf-b4d5a2387ab6.png">

Reply:
Hmm, So strange we don't use `coinPrices` for assertion anymore 🤔 Have you tried with `flowOf(emptyList())` @kaungkhantsoe

Reply:
It is the same @Wadeewee

Reply:
Got it, @kaungkhantsoe and Thanks for rechecking it🙏 Let's keep it then 🚀